CHANGELOG.md

```diff
@@ -8,6 +8,102 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+| :-------- | :-------------: | :------------------: |
+| [Managed Identity Wallet (MIW)](https://github.com/eclipse-tractusx/managed-identity-wallet)     | managed-identity-wallet: [0.5.0](https://github.com/eclipse-tractusx/managed-identity-wallet/releases/tag/v0.5.0) | [v0.5.0](https://github.com/eclipse-tractusx/managed-identity-wallet/releases/tag/v0.5.0) |
+| [Country Risk](https://github.com/eclipse-tractusx/vas-country-risk) | country-risk: [4.0.2](https://github.com/eclipse-tractusx/vas-country-risk/releases/tag/country-risk-4.0.2) | [1.3.1](https://github.com/eclipse-tractusx/vas-country-risk/releases/tag/v1.3.1) |
+
+### Known Knowns
+
+- Relevant components and interfaces developed considering Gaia-X Trust Framework - 22.10
+- Connection to an external Clearing House service for the entire Catena-X ecosystem is required. The Gaia-X compliance service should generally be capable of recognizing all companies that are able to register with the Catena-X Portal.
+Conformity to [Asset Administration Shell (AAS) API v3.0](https://industrialdigitaltwin.org/wp-content/uploads/2023/04/IDTA-01002-3-0_SpecificationAssetAdministrationShell_Part2_API.pdf) and bugfix v3.0.1 is given. Fine granular access control introduced for optional use.
+- This Tractus-X release does not include a Managed Identity Wallet (MIW). The development team decided on a temporary bridging solution, based on a commercial application, to test and ship Release 24.08. Without implementing this interim solution, participants will not be able to fully utilize this Tractus-X release in consequence.
+- For E2E Testing, the [1.2.0](https://github.com/SAP/ssi-dim-middle-layer/releases/tag/v1.2.0) of the [SSI DIM Middle Layer](https://github.com/SAP/ssi-dim-middle-layer) was used - this component is FOSS but the currently used wallet (see previous item) is not.
+- The Tractus-X EDC has undergone significant changes in its Management API, communication with external services and internal APIs.
+- Documentation content (e.g. architecture, manuals etc) and folder structure have not explicitly been checked for this Release, as relevant TRGs are not yet in force. The vast majority of components are evolutions of previous Releases, with a thorough review undergone. This leads to a high confidence in the quality of existing documentation within the product repositories.
+- FOSS components developed and pre-tested to TRL6 with artificial test data only
+- No crosscheck of functionality in verification environment
+- No execution of Load-, Performance- or Penetration Tests
+- No assignment of Export Control Classification Numbers (ECCN) to FOSS components
+- GeoBlocking recommended for Operations (GBaaS)
+- Security concept (overall) recommended for Operations
+- Listed components in "Unchanged, untested" category were not verified in combination with the current Release package and are only compatible with Catena-X Standards as specified. Use at your own risk at this time.
```

Comment:
BPDM has some Known Knowns regarding Portal integration listed in detail here: https://github.com/eclipse-tractusx/bpdm/blob/release/6.1.x/docs/architecture/11_Risks_And_Technical_Debts.md

```
- BPDM standard configuration not compatible with Portal's Central-IDP configuration
- Portal's Partner Network page does not currently show business partner member data due to an authorization issue
- Currently not possible to add company's site and address business partners over the Portal due to authorization issue
- Security concern about exposed technical users when subscribing BPDM services over the Portal
```

Reply:
Thx added your points to the list